automation.py

```diff
@@ -1,17 +1,16 @@
+
 import unittest
-from flask_init import app
+import requests
+
+from flask_init import index
+
+
+
 class Tester(unittest.TestCase):
-	def test_init_server(self):
-		tester = app.test_client(self)
-		response = tester.get('/', content_type='html/text')
-		self.assertEqual(response.status_code, 200)
-		
-	def test_Failure(self):
-		tester = app.test_client(self)
-		response = tester.get('a', content_type='html/text')
-		self.assertEqual(response.status_code, 404)
-		
+    def test_request_response(self):
+    	response = requests.get("http://127.0.0.1:5000")
+        self.assertEqual(response.status_code, 200)
```

Comment:
redundant indentation

Reply:
there is no indentaion in my file, and when i try to edit the file from github the indentation disappears,i will try to push again with the changes sim requested. 

![image](https://user-images.githubusercontent.com/59365386/78894362-90f59f80-7a75-11ea-96ed-ca44ef51e952.png)

Reply:
The problem is a mixture of tabs and spaces, which python doesn't handle well.
On line 11, you have 3 spaces and then a tab.
You can see this by running flake8 against it:
$ flake8 automation.py
automation.py:8:5: E101 indentation contains mixed spaces and tabs
automation.py:8:5: W191 indentation contains tabs